### PR TITLE
Fix peering terms and process

### LINKED ISSUE #2617: Fix terminology for Peering/Attachments for AWS
Hi Team,

Is it correct to call it a `peering attachment` in these <https://docs.astronomer.io/astro/connect-aws?tab=Transit%20Gateways#connection-options|docs>? My understanding is that we create a VPC attachment. <https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html|Peering attachments> seem to refer to connecting two TGWs?

I used that nomenclature in an email/support ticket with a customer as I was referencing our docs. However, they have called this out and it has caused confusion on what is being done. <@U02NP5BKM35> <@U037NU6CMJS> <@U0271C4H71U> - thoughts?

[Slack Message](https://astronomer.slack.com/archives/C015V2JFKT5/p1690408016160169?thread_ts=1690408016.160169&cid=C015V2JFKT5)

### PR DESCRIPTION
**Pull request by manmeetkaur**
This PR clears the steps for creating a creating a new TGW attachment. Resolves #2617 

Reference: 
- https://astronomer.zendesk.com/knowledge/articles/8643574453651/en-us?brand_id=360001488713
- https://astronomer.zendesk.com/agent/tickets/11659

### CODE REVIEW

File: astro/connect-aws.md [hunk 1]
```diff
@@ -102,13 +102,21 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
+2. In your AWS console, [create a resource share in AWS RAM](https://docs.aws.amazon.com/ram/latest/userguide/working-with-sharing-create.html). Then, copy the ID of your existing transit gateway.
+3. Contact [Astronomer support](https://cloud.astronomer.io/support) and provide the following:
+
+    - Cluster **ID** from step 1
+    - Your transit gateway ID from step 2.
+    - Your CIDR block for the target VPC or on-premises network that you want to connect your Astro cluster with
+
+    Then, Astronomer support approves the resource sharing request, attaches the Astro private subnets to your transit gateway, and creates routes in the Astro route tables to your transit gateway for each of the CIDR provided.
+
+4. After you receive confirmation from Asronomer support with the Astro CIDR, you can [create back routes](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-add-route) from the transit gateway to the Astro VPC.
```
Comment (stuart23) [inline astro/connect-aws.md:115]:
```suggestion
4. After you receive confirmation from Astronomer support with the Astro CIDR, you can [create back routes](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-add-route) from the transit gateway to the Astro VPC.
```

File: astro/connect-aws.md [hunk 2]
```diff
@@ -102,13 +102,21 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
-4. Accept the transit gateway peering attachment request from your network. See [Accept or reject a peering attachment request](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-accept-reject).
-5. Create a static route from your CIDR block to the transit gateway and a static route from the transit gateway to the Astro VPC. See [Add a route to the transit gateway route table](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-add-route).
-6. Contact [Astronomer support](https://cloud.astronomer.io/support) to confirm that you have created the static route. Astronomer support will update the Astro VPC routing table to send traffic from your CIDR block through the transit gateway.
-7. Optional. Repeat the steps for each Astro cluster that you want to connect to your transit gateway.
+1. In the Cloud UI, click on the Astronomer icon in the top left corner, go to the **Clusters** tab and copy the **ID** for your Astro cluster.
+2. In your AWS console, [create a resource share in AWS RAM](https://docs.aws.amazon.com/ram/latest/userguide/working-with-sharing-create.html). Then, copy the ID of your existing transit gateway.
```
Comment (manmeetkaur) [inline astro/connect-aws.md:106]: add that it is the same acount where your TGW resides today

**Review by stuart23 — APPROVED** [review]

File: astro/connect-aws.md [hunk 3]
```diff
@@ -102,13 +102,21 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
-4. Accept the transit gateway peering attachment request from your network. See [Accept or reject a peering attachment request](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-accept-reject).
-5. Create a static route from your CIDR block to the transit gateway and a static route from the transit gateway to the Astro VPC. See [Add a route to the transit gateway route table](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-add-route).
-6. Contact [Astronomer support](https://cloud.astronomer.io/support) to confirm that you have created the static route. Astronomer support will update the Astro VPC routing table to send traffic from your CIDR block through the transit gateway.
-7. Optional. Repeat the steps for each Astro cluster that you want to connect to your transit gateway.
+1. In the Cloud UI, click on the Astronomer icon in the top left corner, go to the **Clusters** tab and copy the **ID** for your Astro cluster.
```
Comment (jwitz) [inline astro/connect-aws.md:105]:
```suggestion
1. In the Cloud UI, click the name of your Workspace in the upper left corner of the Cloud UI, then click **Organization Settings** > **Clusters*. Open your cluster from the table that appears and copy its **ID**.
```

File: astro/connect-aws.md [hunk 4]
```diff
@@ -102,13 +102,21 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
+2. Go to your AWS console, and copy the ID of your existing transit gateway (TGW). Then, [create a resource share in AWS RAM](https://docs.aws.amazon.com/ram/latest/userguide/working-with-sharing-create.html) and [follow the AWS documentation](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-transit-gateways.html#tgw-sharing) to share the TGW with the Astro AWS account.
```
Comment (jwitz) [inline astro/connect-aws.md:106]:
```suggestion
2. In your AWS console, copy the ID of your existing transit gateway (TGW). Then, [create a resource share in AWS RAM](https://docs.aws.amazon.com/ram/latest/userguide/working-with-sharing-create.html) and [share the TGW with your Organization's Astro AWS account](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-transit-gateways.html#tgw-sharing).
```
Its the AWS account for the Org, not the cluster right? Where do they find the ID to specify?

Also, where do they need to specify the TGW ID? If they just need it for the Astro support team, we should split this into a second step.

File: astro/connect-aws.md [hunk 5]
```diff
@@ -102,13 +102,21 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
+3. Contact [Astronomer support](https://cloud.astronomer.io/support) and provide the following:
```
Comment (jwitz) [inline astro/connect-aws.md:107]:
```suggestion
3. Contact [Astronomer support](https://cloud.astronomer.io/support) and provide the following information:
```

File: astro/connect-aws.md [hunk 6]
```diff
@@ -102,13 +102,21 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
+
+    - Cluster **ID** from step 1
+    - Your transit gateway ID from step 2.
+    - Your CIDR block for the target VPC or on-premises network that you want to connect your Astro cluster with
```
Comment (jwitz) [inline astro/connect-aws.md:111]:
```suggestion
    - Your cluster **ID** from Step 1.
    - Your TGW ID from Step 2.
    - The CIDR block for the target VPC or on-premises network that you want to connect your Astro cluster with.
```

File: astro/connect-aws.md [hunk 7]
```diff
@@ -102,13 +102,21 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
+
+    Then, Astronomer support approves the resource sharing request, attaches the Astro private subnets to your transit gateway, and creates routes in the Astro route tables to your transit gateway for each of the CIDR provided.
```
Comment (jwitz) [inline astro/connect-aws.md:113]:
```suggestion
    Astronomer support approves the resource sharing request, attaches the Astro private subnets to your transit gateway, and creates routes in the Astro route tables to your transit gateway for each of the CIDRs provided.
```

Comment (jwitz) [reply astro/connect-aws.md:113]: Great description

File: astro/connect-aws.md [hunk 8]
```diff
@@ -102,13 +102,21 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
+
+4. After you receive confirmation from Astronomer support with the Astro CIDR, you can [create back routes](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-add-route) from the transit gateway to the Astro VPC.
```
Comment (jwitz) [inline astro/connect-aws.md:115]:
```suggestion
4. After you receive confirmation from Astronomer support, [create back routes](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-add-route) from the transit gateway to the Astro VPC.
```

File: astro/connect-aws.md [hunk 9]
```diff
@@ -102,13 +102,21 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
+
+6. Contact [Astronomer support](https://cloud.astronomer.io/support) to confirm that you have created the static route. Astronomer support will then test the connection and confirm.
+
+7. (Optional) Repeat the steps for each Astro cluster that you want to connect to your transit gateway.
```
Comment (jwitz) [inline astro/connect-aws.md:119]:
```suggestion
5. Contact [Astronomer support](https://cloud.astronomer.io/support) to confirm that you have created the static route. Astronomer support then tests the connection and confirm.

6. (Optional) Repeat the steps for each Astro cluster that you want to connect to your transit gateway.
```

**Review by jwitz — CHANGES_REQUESTED** [review]
A few questions about step 2, but besides that LGTM!

File: astro/connect-aws.md (hunk 8 above)
Comment (manmeetkaur) [reply astro/connect-aws.md:115]: Astronomer support will provide the Astro CIDR, so we need to include that. let me try to reframe the sentence.

**Comment by manmeetkaur** [comment]
@jwitz you can review again. Changes include:

- split the step #2 into 2 steps. 
- added info about Astro CIDR to the para in step 3.

File: astro/connect-aws.md [hunk 10]
```diff
@@ -102,13 +102,22 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
-6. Contact [Astronomer support](https://cloud.astronomer.io/support) to confirm that you have created the static route. Astronomer support will update the Astro VPC routing table to send traffic from your CIDR block through the transit gateway.
-7. Optional. Repeat the steps for each Astro cluster that you want to connect to your transit gateway.
+1. In the Cloud UI, click the name of your Workspace in the upper left corner of the Cloud UI, then click **Organization Settings** > **Clusters*. Open your cluster from the table that appears and copy its **ID**.
+2. In your AWS console, copy the ID of your existing transit gateway (TGW). 
+3. [Create a resource share in AWS RAM](https://docs.aws.amazon.com/ram/latest/userguide/working-with-sharing-create.html) and [share the TGW with your cluster's Astro AWS account](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-transit-gateways.html#tgw-sharing). If you do not have your Astro AWS Account, reach out to [Astronomer support](https://cloud.astronomer.io/support).
```
Comment (jwitz) [inline astro/connect-aws.md:107]:
```suggestion
3. [Create a resource share in AWS RAM](https://docs.aws.amazon.com/ram/latest/userguide/working-with-sharing-create.html) and [share the TGW with your cluster's Astro AWS account](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-transit-gateways.html#tgw-sharing). 

    To find your cluster's AWS account ID in the Cloud UI, click the name of your Workspace in the upper left corner, click **Organization Settings**, then click **Clusters**. Open your cluster and copy its **Account ID**.
```

File: astro/connect-aws.md [hunk 11]
```diff
@@ -102,13 +102,22 @@ If Astronomer creates a new transit gateway in your AWS account for Astro, keep
...
+    - Your TGW ID from Step 2.
+    - The CIDR block for the target VPC or on-premises network that you want to connect your Astro cluster with.
+
+    Astronomer support approves the resource sharing request, attaches the Astro private subnets to your transit gateway, and creates routes in the Astro route tables to your transit gateway for each of the CIDR provided. Astronomer support notifies you about the process completion and provides you with the Astro CIDRs.
+
+4. After you receive the confirmation from Astronomer support, you can [create back routes](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-add-route) from the transit gateway to the Astro VPC.
+
+5. Contact [Astronomer support](https://cloud.astronomer.io/support) to confirm that you have created the static route. Astronomer support then tests the connection and confirm.
+
+6. (Optional) Repeat the steps for each Astro cluster that you want to connect to your transit gateway.
```
Comment (jwitz) [inline astro/connect-aws.md:120]:
```suggestion
5. After you receive the confirmation from Astronomer support, you can [create back routes](https://docs.aws.amazon.com/vpc/latest/tgw/tgw-peering.html#tgw-peering-add-route) from the transit gateway to the Astro VPC.

6. Contact [Astronomer support](https://cloud.astronomer.io/support) to confirm that you have created the static route. Astronomer support then tests the connection and confirm.

7. (Optional) Repeat the steps for each Astro cluster that you want to connect to your transit gateway.
```

File: astro/connect-aws.md (hunk 10 above)
Comment (jwitz) [reply astro/connect-aws.md:107]: Feel free to amend if this is not correct

**Review by jwitz — APPROVED** [review]

File: astro/connect-aws.md (hunk 10 above)
Comment (manmeetkaur) [reply astro/connect-aws.md:107]: @jwitz unfortunately, Account ID is not visible in the Cloud UI. 
Screenshot 2023-08-09 at 9 09 54 PM

Comment (jwitz) [reply astro/connect-aws.md:107]: Oh my mistake, you can see it on Hybrid but not on Hosted.
Screenshot 2023-08-10 at 11 05 26 AM

Updated my suggestion to reflect this

**Review by jwitz — APPROVED** [review]

**Pull request merged**